bHY6yh-d1e515/cod-bxxqqq_bHY6yh-d1e515.xml

```diff
@@ -73,37 +73,40 @@
           <lb ed="#B"/>et paciens</quote>
         </p>
         <p xml:id="bHY6yh-d1e515-d1e142"> /
-           <ref xml:id="bHY6yh-d1e515-Rd1e142"> psalmus. 17. </ref> <quote xml:id="bHY6yh-d1e515-Qd1e144"> Protector est deus
-            omnium sperantium in se. </quote>
+           <ref xml:id="bHY6yh-d1e515-Rd1e142"> psalmus 17</ref> <quote xml:id="bHY6yh-d1e515-Qd1e144"> Protector est 
+           <lb ed="#B"/>deus sperantium in se. </quote>
         </p>
         <p xml:id="bHY6yh-d1e515-d1e150"> /
-          <ref xml:id="bHY6yh-d1e515-Rd1e152">psalmus. 24</ref> <quote xml:id="bHY6yh-d1e515-Qd1e152">
-               Fundamentum est dominus omnibus timentium eum. </quote>
+          <ref xml:id="bHY6yh-d1e515-Rd1e152">psalmus 24</ref> <quote xml:id="bHY6yh-d1e515-Qd1e152"> Fun
+               damentum est dominus omnibus timen
+               <lb ed="#B"/>tibus eum </quote>
         </p>
         <p xml:id="bHY6yh-d1e515-d1e158"> /
-          <ref xml:id="bHY6yh-d1e515-Rd1e162">psalmus. 33.</ref> 
-            <quote xml:id="bHY6yh-d1e515-Qd1e160">Gustate et videte quoniam suauis est dominus.</quote>
+          <ref xml:id="bHY6yh-d1e515-Rd1e162">psalmus 33</ref> 
+            <quote xml:id="bHY6yh-d1e515-Qd1e160">Gustate et vi
+            <lb ed="#B"/>dete quoniam suauis est dominus.</quote>
```

Comment:
one note: If a line break falls in the middle of the a word, we want to place the line break directly in the middle of the word, then let the word finish, and then hit hard return to begin the next line

So in this example here with "videte" we should have the following: 

```
Gustate et vi<lb ed="#B"/>dete 
quoniam suauis est domins.
```